assets/js/users.js

```diff
@@ -1,11 +1,33 @@
-/* global userRepository */
-
-function User(firstName, lastName, emailAddress, username, password, avatar) {
-  this.id = 1;
+function User(id, firstName, lastName, emailAddress, username, password, avatar) {
+  this.id = id;
   this.firstName = firstName;
   this.lastName = lastName;
   this.emailAddress = emailAddress;
   this.username = username;
   this.password = password;
   this.avatar = avatar;
 }
+
+fetch('https://jsonplaceholder.typicode.com/users')
```

Comment:
Please check my Weather Dashboard app and see how I tried to handle errors in fetch method. 

What happens if there is an error with the API?